gheb/docusign-bundle/4.5/config/packages/docusign.yaml

```diff
@@ -0,0 +1,17 @@
+docusign:
+    access_token: "YourAccessToken"
```

Comment:
Please add a comment with the link where to generate the token: https://developers.docusign.com/oauth-token-generator